Cargo.toml

```diff
@@ -105,7 +105,7 @@ features = [
 wayland-client = { version = "0.29.4", default_features = false,  features = ["use_system_lib"], optional = true }
 wayland-protocols = { version = "0.29.4", features = [ "staging_protocols"], optional = true }
 sctk = { package = "smithay-client-toolkit", version = "0.16.0", default_features = false, features = ["calloop"],  optional = true }
-sctk-adwaita = { version = "0.4.1", optional = true }
+sctk-adwaita = { version = "0.5.0", optional = true }
```

Comment:
Please, specify the latest versions.

Reply:
Why though? What's wrong with users being able to downgrade if they want to?

Anyway, done.

Reply:
Because it's the latest version you're writing the patch against? So we know that it works, but we don't know that `0.5.0` works. In general you should use the dependency you're writing the patch against.

Reply:
I was actually writing the patch against 0.5.0, not 0.5.1. In any case, I really don't have a stack in this, I was going to use the newest version anyway.